disnake/channel.py

```diff
@@ -100,7 +102,87 @@ async def _single_delete_strategy(messages: Iterable[Message]) -> None:
         await m.delete()
 
 
-class TextChannel(disnake.abc.Messageable, disnake.abc.GuildChannel, Hashable):
+class WebhookableChannel:
```

Comment:
this should be a separate PR

Reply:
reverted in 3e5ed18f8d5c4044da4fd0f1ee41282d1cf81c6c for now. Still, refactors like that would be great as new PRs